squash_wip.go

```diff
@@ -32,13 +24,24 @@ last commit must be a manual commit`)
 		mobExecutable()+" squash-wip --git-editor",
 		mobExecutable()+" squash-wip --git-sequence-editor",
 	)
-	silentgit("rebase", "-i", "--keep-empty", mergeBase)
+	git("rebase", "-i", "--keep-empty", mergeBase)
```

Comment:
Instead of "-i" we should write "--interactive" (always use the more readable parameter)